help/destinations/catalog/streaming/pegaCDH.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Pega Customer Decision Hub connection
+description: Use the Pega Customer Decision Hub destination in Adobe Experience Platform to send profile attributes and segment membership data to Pega Customer Decision Hub for next-best-action decisioning.
+---
+
+# Pega Customer Decision Hub connection
+
+## Overview {#overview}
+
```

Comment:
In the overview, please add a disclaimer note, specifying that this documentation page was created by the Pega team, and sharing a support channel: 

```
>[!IMPORTANT]
>
>This documentation page was created by the *YourDestination* team. For any inquiries or update requests, please contact them directly at *Insert link or email address where you can be reached for updates, for example `support@YourDestination.com`.*
```

You can view an example in the partner-authored Snap destination: https://experienceleague.adobe.com/docs/experience-platform/destinations/catalog/advertising/snap-inc.html?lang=en